cli/src/main.rs

```diff
@@ -463,6 +483,43 @@ async fn run_cache(command: &CacheCommand) -> Result<(), CliError> {
     Ok(())
 }
 
+async fn run_init(args: &InitArgs) -> Result<(), CliError> {
+    match args.language.to_lowercase().trim() {
+        "rust" => {
+            println!(
+                "{:?}",
+                std::process::Command::new("git")
+                    .arg("clone")
+                    .arg(RUST_TEMPLATE)
+                    .output()
+            );
+        }
+        "c++" | "cpp" => {
+            println!(
+                "{:?}",
+                std::process::Command::new("git")
+                    .arg("clone")
+                    .arg(CPP_TEMPLATE)
+                    .output()
+            );
+        }
+        "c" => {
+            println!(
+                "{:?}",
+                std::process::Command::new("git")
+                    .arg("clone")
+                    .arg(C_TEMPLATE)
+                    .output()
+            );
+        }
+        _ => return Err(CliError::Cache),
```

Comment:
Maybe not `CliError::Cache`? We should probably do a custom error type, like `CliError::TemplateTag("swift")` to get the error `Template matching "swift" doesn't exist`

Reply:
Oops, youre right, this was a placeholder heh